tests/smoke/Makefile

```diff
@@ -1,4 +1,12 @@
-test:
+SHELL:=/bin/bash
+
+deps:
+	git clone https://github.com/sstephenson/bats.git
+	mkdir ~/bin || :
```

Comment:
I see, makes sense. 

Not that this is a "dirty" solution or anything,  but this makes me think we should start carving out a common pattern for "setting things up for CI" vs "setting things up locally" in our Makefiles. Hopefilly, its very clear for future maintainers "hey, this is slightly odd because it's running in CI, not necessarily your local environment"